labelbox/schema/bulk_import_request.py

```diff
@@ -180,12 +183,39 @@ def get(cls, client: Client, project_id: str, name: str) -> 'BulkImportRequest':
 
     def refresh(self) -> None:
         """
-       Synchronizes values of all fields with the database.
-       """
+        Synchronizes values of all fields with the database.
+        """
         bulk_import_request = self.get(self.client, self.project().uid, self.name)
         for field in self.fields():
             setattr(self, field.name, getattr(bulk_import_request, field.name))
 
+    def wait_until_done(
+            self, sleep_time_seconds: int = 30) -> None:
```

Comment:
where does `sleep_time_seconds` get used?  i only see it in the log line

Reply:
Whoops. Good catch!